packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -60,6 +60,7 @@ export const Block = Node.create<IBlock>({
     return {
       listType: {
         default: undefined,
+        // Causes nesting issues pasting outside editor
```

Comment:
can you explain this / is this still accurate?

Reply:
Yeah, lists copied from BlockNote don't get formatted properly in other apps - I'm pretty sure it's down to the fact that we don't use _ul_ or _ol_ tags to mark the start and end of a list. Copying a list from BlockNote and pasting it in e.g. Google Docs therefore makes list  items get reformatted as paragraphs, so nesting also doesn't work. Not sure if this is currently relevant right now though.